modules/nested-elements/assets/js/editor/nested-repeater/views/widget-repeater/empty.js

```diff
@@ -0,0 +1,91 @@
+import { useRef, useState } from 'react';
+
+export default function Empty( props ) {
+	// On container remove.
+	if ( props.container.view.isDisconnected() ) {
+		return null;
```

Comment:
Set a state with the isConnected status, remove the return. It's recommended to have only one return statement

Reply:
Handled.